packages/aws-cdk-lib/aws-stepfunctions/lib/states/state.ts

```diff
@@ -491,13 +491,26 @@ interface ChoiceTransition {
   condition: Condition;
 }
 
+/**
+ * Options for Choice Transition
+ */
+export interface IChoiceTransitionOptions {
+  /**
+   * An optional description for the choice transition
+   *
+   * @default No comment
+   */
+  comment?: string;
```

Comment:
does this fix the JSII error maybe?

```suggestion
  readonly comment?: string;
```

Reply:
@jogold It did. Good shout, Jonathan. I was following the pattern of `state` and `condition` but there is no reason this can't be `readonly`.